client/src/components/Reviews/Form.jsx

```diff
@@ -1,90 +1,146 @@
 import React, { useState, useEffect } from 'react';
-import { FormView } from '../styles/reviewstyles/FormView.styled.js';
-import { Background } from '../styles/reviewstyles/FormBackground.styled.js';
-import { SummaryBox } from '../styles/reviewstyles/SummaryBox.styled.js';
+import { ThumbNail, FullSize } from '../styles/reviewstyles/imageStyles.styled.js';
+import { FormModal, FormModalBackground, Star, FormTitle, FormGrid, FormRow, FormCol } from '../styles/reviewstyles/formStyles.styled.js';
+import axios from 'axios';
 
-const Form = ({ closeForm, form }) => {
+import StyleRating from './StyleRating.jsx';
+import StarRating from './StarRating.jsx';
+
+const Form = ({ closeForm, form, metaReview, currentProductId, }) => {
+  //form values
   const [summary, setSummary] = useState('');
   const [name, setName] = useState('');
   const [review, setReview] = useState('');
   const [email, setEmail] = useState('');
-  const [rating, setRating] = useState(null);
-  const [recommend, setrecommend] = useState(false);
-
-  const [maxFiles, setFiles] = useState(0);
+  const [recommend, setRecommend] = useState(null);
+  const [photoArray, setPhotoArray] = useState('');
+  //star rating values
```

Comment:
I haven't tried useReduce, but it could be a good option to handle many states

Reply:
Will look into during refactoring process. Could be great for my form hooks